learning/tour-of-beam/frontend/lib/pages/tour/state.dart

```diff
@@ -53,6 +56,14 @@ class TourNotifier extends ChangeNotifier with PageStateMixin<void> {
         treeIds: contentTreeController.treeIds,
       );
 
+  void _setSdk() {
```

Comment:
`_onAppNotifierChanged`
`_setSdk` gives an impression that it receives the changed value as an argument.